### DISCUSSION TITLE
Command Palette don't work with Russian keyboard layout

### DISCUSSION THREAD

**Discussion by bagatka** [community/community#6897] [Other Feature Feedback, Questions, & Ideas] [answered]
As a native Russian-speaker, I do not always have the English keyboard layout selected in the operating system, therefore, I am limited in the use of "Command Palette", I need to switch to the English keyboard layout every time. I'm not sure, but it can effect some other language layouts with different symbols too

**Comment by gaknoll** [comment, marked as answer]
Sorry about the inconvenience here @bagatka! We've gotten a lot of feedback on the default keyboard shortcuts used to open the command palette and will be looking into solutions that will help users like yourself more easily use this feature.

**Reply by bagatka** [reply]
Thank you @gaknoll. Looking forward!
